Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -4673,5 +4673,17 @@
   {
     "Input": "+1,032.923 is not a phone number",
     "Results": []
+  },
+  {
+    "Input": "0\t0\t0\t0\t0",
```

Comment:
You can leave only one such case with tabs. What about one that has enough 0 to be a phone number?